tap/extensions/amqp/main.go

```diff
@@ -78,15 +78,17 @@ func (d dissecting) Dissect(b *bufio.Reader, isClient bool, tcpID *api.TcpID, co
 	var lastMethodFrameMessage Message
 
 	for {
+		// fmt.Printf("lastMethodFrameMessage: %v\n", lastMethodFrameMessage)
 		frame, err := r.ReadFrame()
 		if err == io.EOF {
 			// We must read until we see an EOF... very important!
 			return errors.New("AMQP EOF")
 		} else if err != nil {
-			// TODO: Causes ignoring some methods. Return only in case of a certain error. But what?
-			return err
+			// 	return err
```

Comment:
Was this line commented out on purpose?
If it was, then remove the `else` clause.

Reply:
@nimrod-up9 yes it was intentional.

The compiler probably ignores empty `else if` outcomes but removed it with https://github.com/up9inc/mizu/commit/1a6d32e6614e4587883b27dda599f011054b8605 anyway.